<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="workArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>751</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="New_Game">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>50</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>NEW GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="Prompt">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start New Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="Stopwatch">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>270</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Mode">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>350</y>
      <width>140</width>
      <height>143</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Mode_1">
       <property name="text">
        <string>Unbound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Mode_2">
       <property name="text">
        <string>Jailed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Mode_3">
       <property name="text">
        <string>Trick O' Treat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="Difficulty">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>520</y>
      <width>140</width>
      <height>143</height>
     </rect>
    </property>
    <property name="title">
     <string>Difficulty</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="Easy">
       <property name="text">
        <string>Easy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Medium">
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Hard">
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Bomb">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>220</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bomb</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
